CleverTapSDK/Encryption/CTCryptMigrator.m

```diff
@@ -0,0 +1,307 @@
+#import <UIKit/UIKit.h>
+#import "CTLocalDataStore.h"
+#import "CTConstants.h"
+#import "CTPreferences.h"
+#import "CTLoginInfoProvider.h"
+#import "CTEncryptionManager.h"
+#import "CTPreferences.h"
+#import "CTUtils.h"
+#import "CTCryptMigrator.h"
+
+NSString *const kCachedGUIDSKey = @"CachedGUIDS";
```

Comment:
The code for getting the cached GUIDs is duplicated from the `CTLoginInfoProvider`, please check if this can be optimised.